src/common/models/Cookie.class.js

```diff
@@ -766,6 +756,32 @@ class Cookie {
     return `${h}:${m}`;
   }
 
+  parseVoyageObj(type, voyageObj) {
+    let voyage = {};
```

Comment:
Use const not let as it is not being reassigned

Reply:
Sure, it is done.

 I guess when I was originally doing it, I though that using a `const` for a variable whose entire purpose to mutated seem a bit a weird, but I am also happy for the changes.